README.md

```diff
@@ -1,2 +1,26 @@
-# IC_Test
-This repository holds all of the research and code used to develop IC Test.
+# Microscopy Imaging Illumination and Contrast Evaluation
+
+![MIICE logo](./media/logo/MIICE_logo_v1.png)
+
+> Don’t let your images squeak by untested! 🐭🧀
+
+MIICE is a Python package to evaluate microscopy images for quality issues like uneven illumination and poor contrast prior to perform image analysis.
+
+It is nearly impossible to acquire "perfect" images across large high-throughput screens.
+Technical artifacts from the microscope can impact all of the images (like uneven illumination) or some of the images (out-of-focus, over-saturation, or poor contrast).
+Most modern microscopes have the ability to try and correct for this, but it is not always full proof.
```

Comment:
Small suggestion: It might be more effective to introduce the problem first and then present MIICE as the solution. This paragraph is strong and engaging, so placing it at the beginning could help draw the reader in right away.

Reply:
This is a great suggestion! I will implement that now! Thanks Erik!